src/auction.mo

```diff
@@ -762,19 +758,9 @@ module {
 
     var nextAssetIdToProcess : Nat = 0;
 
-    public func onTimer() : async () {
-      if (nextAssetIdToProcess == 0) {
-        switch (hooks.preAuction) {
-          case (?hook) await* hook();
-          case (_) {};
-        };
-      };
+    public func runAuction() : async () {
```

Comment:
This function should not be `async` return type.

`processAssetsChunk` does not need to be `async*`.

Reply:
Done